src/include/homeobject/blob_manager.hpp

```diff
@@ -10,7 +10,7 @@
 namespace homeobject {
 
 ENUM(BlobError, uint16_t, UNKNOWN = 1, TIMEOUT, INVALID_ARG, UNSUPPORTED_OP, NOT_LEADER, REPLICATION_ERROR,
-     UNKNOWN_SHARD, UNKNOWN_BLOB, CHECKSUM_MISMATCH, READ_FAILED, INDEX_ERROR, SEALED_SHARD);
+     UNKNOWN_SHARD, UNKNOWN_BLOB, CHECKSUM_MISMATCH, READ_FAILED, INDEX_ERROR, SEALED_SHARD, DATA_EXIST_ALREADY);
```

Comment:
It's unfortunate that we need to update the public API for an ec that will never leave the library itself... Not sure how to avoid this other than re-use an existing ec incorrectly.../shrug

Reply:
I see your point, let me change the signature of add_index_table() to return internal status and return the actual error as INDEX_ERROR and remove the DATA_EXIST_ALREADY.